### PR TITLE
CUMULUS-3432/CUMULUS-3506 - Upgrade to Aurora PG v13

### PR DESCRIPTION
**Pull request by botanical**
**Summary:** Summary of changes

Addresses [CUMULUS-3432: Upgrade to Aurora PG v13](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3432)
[CUMULUS-3506: Provide instructions to users for RDS module for engine upgrade](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3506)
## Changes

* Updated defaults to use v13

## PR Checklist

- [x] Update CHANGELOG
- [ ] Unit tests
- [x] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: example/spec/parallel/ingestGranule/IngestGranuleFailureSpec.js [hunk 1]
```diff
@@ -252,7 +252,7 @@ describe('The Ingest Granule failure workflow', () => {
         const isSchemaValidationError = (error.Error === 'CumulusMessageAdapterExecutionError') &&
           error.Cause.includes('jsonschema.exceptions.ValidationError');
         const isPostgresWriteError = error.Error.includes('Failed writing files to PostgreSQL') &&
-          error.Cause.includes('null value in column "bucket" violates not-null constraint');
+          error.Cause.includes('null value in column "bucket" of relation "files" violates not-null constraint');
```
Comment (Jkovarik) [inline example/spec/parallel/ingestGranule/IngestGranuleFailureSpec.js:255]: This change was needed due to the enginerupdate

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,39 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+From this release forward, Cumulus Core will be tested against PostgreSQL v13. Users
+should migrate their datastores to Aurora PostgreSQL 13.9+ compatible data
+stores as soon as possible after upgrading to this release.
+
+#### Engine Upgrade
+
+Users utilizing the `cumulus-rds-tf` module will have upgraded/had their
+database clusters forcibly upgraded at the next maintenance window after February 29, 2024.
+
+To upgrade your engine version, we recommend a manual (outside of
```
Comment (npauzenga) [inline CHANGELOG.md:20]: So this is if you don't use the cumulus-rds-tf module? Or are we saying that all users should proceed with the manual upgrade to get ahead of AWS forcing the upgrade in Feb?

File: example/spec/parallel/ingestGranule/IngestGranuleFailureSpec.js (hunk 1 above)
Comment (npauzenga) [reply example/spec/parallel/ingestGranule/IngestGranuleFailureSpec.js:255]: Yeah, probably shouldn't make assertions against the error.Cause in this case if that's a description that might change but not a huge deal.

**Review by npauzenga — APPROVED** [review]
Looks good to me! Just a couple clarifying questions. Also I see we call out a specific version in the RDS Phase 3 upgrade notes. Hopefully everyone is already well beyond v16 but maybe it's worth updating?

```
The PostgreSQL database cluster should be updated to the supported version (Aurora Postgres 11.13+ compatible)
```
in `upgrade_rds_phase_3_release.md`.

That was the only reference I saw that wasn't updated (likely intentionally).

File: CHANGELOG.md (hunk 2 above)
Comment (Jkovarik) [reply CHANGELOG.md:20]: This is for users utilizing `cumulus-rds-tf`.     Users *not* doing so are on their own with respect to how they update their cluster.

**Pull request merged**